feat: adds --watch to introspect commands (#1207)
apollographql/apollo-cli: docs/source/commands/graphs.mdx

The server must be reachable by Rover and it must have introspection enabled.

> #### Using the `--watch` flag

Reviewer: ```suggestion
#### Watching for schema changes
```

> You can pass `--watch` to `rover graph introspect` and Rover will introspect your GraphQL endpoint every second. Rover will only print an updated introspection response (or error message) to your terminal if the output of the query differs from the output of the previous query. 

Reviewer: ```suggestion
If you pass `--watch` to `rover graph introspect`, Rover introspects your GraphQL endpoint every second. Whenever the returned schema differs from the _previously_ returned schema, Rover outputs the updated schema.  
```
